control-plane/roles/isolated-clusters/templates/ntp/chrony-deployment.yaml

```
@@ -0,0 +1,94 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: chrony
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    k8s-app: chrony
```

Please use the recommended labels, see https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels
```suggestion
    app.kubernetes.io/name: chrony
```